client/src/components/core/ms-modal/utils.ts

```diff
@@ -8,7 +8,14 @@ import FolderSelectionModal from '@/components/core/ms-modal/FolderSelectionModa
 import { FsPath } from '@/parsec';
 import { FolderSelectionOptions, GetPasswordOptions, GetTextOptions, Answer, MsModalResult } from '@/components/core/ms-modal/types';
 
-export async function askQuestion(title: string, subtitle?: string, redisplayMainModalOnYes = true): Promise<Answer> {
+export interface QuestionOptions {
+  yesText?: string;
+  noText?: string;
+  keepMainModalHiddenOnYes?: boolean;
```

Comment:
Renamed this variable from `redisplayMainModalOnYes` because it's easier to have a default "false-ish" value (missing variable counts as false).